src/components/TopBar/index.tsx

```diff
@@ -0,0 +1,126 @@
+import { Slider, SliderTrack, SliderFilledTrack, SliderThumb } from "@chakra-ui/react";
+import { Flex, SimpleGrid, Text, IconButton } from "@chakra-ui/react";
+import {
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverBody,
+  PopoverArrow,
+} from "@chakra-ui/react";
+
+import ColorPlate from "./colorPlate";
+
+import BinSvg from "@/assets/svg/TopBarSvg/bin.svg";
+import DocSvg from "@/assets/svg/TopBarSvg/document.svg";
+import RbSvg from "@/assets/svg/TopBarSvg/rainbow.svg";
+import ScSvg from "@/assets/svg/TopBarSvg/smallCourt.svg";
+import UploadSvg from "@/assets/svg/TopBarSvg/upload.svg";
+
+const TopBar = () => {
+  return (
+    <SimpleGrid
+      columns={2}
+      position="fixed"
+      w="calc(100vw - 98px)"
+      bg="white"
+      boxShadow="dark-lg"
+      left="98px"
+      mt="-356"
```

Comment:
Use top instead of margin-top for locate this component, cos you have fixed position already.

Reply:
thanks tao zong, we have fixed it